src/util.ts

```diff
@@ -15,10 +15,10 @@
  */
 
 import * as fs from 'fs';
-import {access, read} from 'fs/promises';
 import * as path from 'path';
 import pify from 'pify';
 import rimraf from 'rimraf';
+import {file} from 'tmp';
```

Comment:
Because `file` is so broad in its meaning, I would suggest `import * as tmp` so as to be more specific. `tmp.file(...)` is more meaningful than `file(...)`

Reply:
This module isn't used in this file so I removed it